### ISSUE TITLE
Implement https access to MassBank

### ISSUE THREAD

**Issue by tsufz** [MassBank/MassBank-web#23]
Deployment of https services on MassBank server; needs a self-certificate for test case and implementation in Apache configuration

**Comment by sneumann** [comment]
another alternative certificate source is https://letsencrypt.org/

**Comment by tsufz** [comment]
I suggest to implement letsencrypt. Will be implemented after deployment of new server, nice example:
https://kofler.info/lets-encrypt-zertifikate-fuer-web-und-mail-unter-ubuntu-16-04/

**Comment by tsufz** [comment]
@meier-rene @Treutler 
I am working on the https: implementation. The certificate is installed on massbank.eu; you cannot reach by now because port 443 is not yet opened in the FW; but I can play from intranet. I deployed also the letsencrypt certificate into the keyring (https://stackoverflow.com/questions/34110426/does-java-support-lets-encrypt-certificates/38312489#38312489). Hence, the jars are working.

What is not working is the display of the mass spectra and some elements are shown as insecure.

I just searched in the code for http: elements. Lots are external codes. But there are some internal ones. 

I suggest to add alternative URL for https: or if feasible to replace all hard encoded URLs by relative ones to the server root such as ./MassBank/XYZ.
